Automattic/wp-calypso
# Move tags to new line and update styling

#### Proposed Changes

* Moves reader tags to a new line to allow for new styling

#### Testing Instructions

* Create a post (that you follow on reader) with more than 3 tags and see how this looks on reader. It should match styling below;

![Image](https://user-images.githubusercontent.com/5634774/184981937-5bc27b4d-b297-49ce-ac4c-2e42d314037b.png)

Related to https://github.com/Automattic/wp-calypso/issues/66622
